lib/resque/plugins/retry.rb

```diff
@@ -388,9 +398,22 @@ def try_again(exception, *args)
 
         # some plugins define retry_delay and have it take no arguments, so rather than break those,
         # we'll just check here to see whether it takes the additional exception class argument or not
-        temp_retry_delay = ([-1, 1].include?(method(:retry_delay).arity) ? retry_delay(exception.class) : retry_delay)
+        # we also allow all job args to be passed to a custom `retry_delay` method
+        retry_delay_arity = method(:retry_delay).arity
+
+        temp_retry_delay = if [-2, 2].include?(retry_delay_arity)
+          retry_delay(exception.class, *args)
+        elsif [-1, 1].include?(retry_delay_arity)
+          retry_delay(exception.class)
+        else
+          retry_delay
+        end
+
+        retry_job_class = retry_job_delegate ? retry_job_delegate : self
+
+        retry_in_queue = retry_queue(exception, *args)
```

Comment:
Please 1-line these.

Reply:
@jzaleski do you want me to 1-line the `temp_retry_delay` assignment? Which line exactly?

Reply:
@jzaleski ^ quick ping on the above comment!

Reply:
@jzaleski which line exactly do you want me to one-line here?

Reply:
It looks as though the commit I was referring to previous is now gone. I'm going to drop this.